core/src/kura.rs

```diff
@@ -328,6 +347,35 @@ impl Kura {
         data.pop();
         data.push((block.hash(), Some(block)));
     }
+
+    /// Store WSV to disk.
+    pub fn store_wsv(&self, wsv: &WorldStateView) {
+        let data: WSVData = wsv.into();
+        let path = self.block_store_path.join(WSV_FILE_NAME);
+
+        std::thread::spawn(move || {
+            let bin_data: Vec<u8> = data.encode();
+
+            let mut wsv_file = std::fs::OpenOptions::new()
+                .write(true)
+                .truncate(true)
+                .create(true)
+                .open(path.clone())
+                .expect("Failed to open wsv file");
+
+            wsv_file
+                .write_all(&bin_data)
+                .expect("Failed to write wsv file");
```

Comment:
Please, specify a path to the file in panics

Reply:
Or don't panic. This looks like an error that could be forwarded up and handled later.